package.json

```diff
@@ -24,6 +24,10 @@
         "coveralls": "cat ./coverage/lcov.info | node node_modules/.bin/coveralls",
         "release": "standard-version -t ''"
     },
+    "engines": {
+        "node": ">=12",
+        "npm": ">=7.1.0"
```

Comment:
Node 7 is required to generate a v2 lockfile. Node 15 is the first version that ships with npm 7. So shouldn't we require node >= 15 rather than 12?